index.html

```diff
@@ -1,14 +1,158 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Asia Web Conference 2022</title>
   <link rel="stylesheet" href="./style.css">
-  <script src="./script.js"></script>
+  <link rel="shortcut icon" href="./assets/AsiaWebConf-Icon.png" type="image/x-icon">
+  <script src="https://kit.fontawesome.com/41c9961a42.js" crossorigin="anonymous"></script>
+  <script src="./script.js" defer></script>
 </head>
+
 <body>
-  
+  <!-- Heading Section -->
+  <div class="bgContainer" id="home">
+    <div class="menuBar">
+      <button class="menu" type="button">
+        <img src="./assets/hamburger.svg" alt="">
```

Comment:
- [ ] Please, consider filling everywhere the `alt` for each image in both `Index` and `About` pages

Reply:
@mavericks-db In general it should be the alternative text for any image on your projects because as you know, the visitor should know what is the image there if it is not loaded for any reason 👍

Reply:
@elmar8287 Noted. Thanks 👍